services/clojars/clojars-release.service.js

```diff
@@ -0,0 +1,71 @@
+'use strict'
+
+const Joi = require('joi')
+const { version: versionColor } = require('../color-formatters')
+const { BaseJsonService, NotFound } = require('..')
+const { InvalidResponse } = require('../errors');
```

Comment:
`InvalidResponse` can be grabbed from index too, just like `BaseJsonService` and `NotFound`

Reply:
already on it!

Reply:
👍